test/libsolidity/util/BytesUtils.cpp

```diff
@@ -33,6 +35,41 @@ using namespace dev::solidity::test;
 using namespace std;
 using namespace soltest;
 
+bytes BytesUtils::alignLeft(bytes _bytes)
+{
+	soltestAssert(_bytes.size() <= 32, "");
+	return std::move(_bytes) + bytes(32 - _bytes.size(), 0);
+}
+
+bytes BytesUtils::alignRight(bytes _bytes)
+{
+	soltestAssert(_bytes.size() <= 32, "");
+	return bytes(32 - _bytes.size(), 0) + std::move(_bytes);
+}
+
+bytes BytesUtils::applyAlign(
+	Parameter::Alignment _alignment,
+	ABIType& _abiType,
+	bytes _bytes
+)
+{
+	if (_alignment != Parameter::Alignment::None)
+		_abiType.alignDeclared = true;
+
+	switch (_alignment)
+	{
+	case Parameter::Alignment::Left:
+		_abiType.align = ABIType::AlignLeft;
+		return alignLeft(std::move(_bytes));
+	case Parameter::Alignment::Right:
+		_abiType.align = ABIType::AlignRight;
```

Comment:
These two lines can be removed

Reply:
Maybe a case for `[[fallthrough]]` (in the future)?